Adds APIM pages docs (#3868)
wso2/docs-apim: en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-overview.md

#Overview Page
![overview page full]({{base_path}}/assets/img/observe/overview/overview-page-full.png)

> This is the welcome page of the analytics portal. This page will mainly display a quick overview of the whole

Reviewer: ```suggestion
This is the welcome page of the analytics portal. This page gives you a quick overview of the whole
```

>  management system. Targeted audience for the page are managers and business users who need quick insight. Also this

Reviewer: ```suggestion
 management system. Targeted audience for the page are managers and business users who need quick insight. Also, this
```

>   page can be used as a dashboard displaying current system status.

Reviewer: ```suggestion
  page can be used as a dashboard to view the current system status.
```

#Overview Page
![overview page full]({{base_path}}/assets/img/observe/overview/overview-page-full.png)
This is the welcome page of the analytics portal. This page will mainly display a quick overview of the whole
 management system. Targeted audience for the page are managers and business users who need quick insight. Also this
  page can be used as a dashboard displaying current system status.
 

> ## Page Walk Through

Reviewer: ```suggestion
```

### Total Traffic Widget
![overview page total traffic]({{base_path}}/assets/img/observe/overview/overview-page-total-traffic.png)

Total traffic widget displays the total traffic your selected environment received during the selected time range

> . This will include both successful requests and error requests. If you want to investigate further on the traffic

Reviewer: ```suggestion
. This includes both successful requests and error requests. If you want to investigate further on the traffic
```

>  please move on to 

Reviewer: ```suggestion
```

>  [traffic page]({{base_path}}/observe/api-manager-analytics/analytics-pages/analytics-pages-traffic).

Reviewer: ```suggestion
```

 
### Latency Widget
![overview page latency]({{base_path}}/assets/img/observe/overview/overview-page-latency.png)

Latency widget displays the 95th percentile of all API latencies in your selected environment for the selected time

>  period. You can utilize this widget to know whether the whole system operates under given SLAs. This metric

Reviewer: ```suggestion
 period. You can use this widget to know whether the whole system operates under given SLAs. This metric
```

>   will give the first indication into whether there are any slow APIs. To investigate further please visit 

Reviewer: ```suggestion
 gives the first indication of slow APIs. To investigate further, see the 
```

  [latency page]({{base_path}}/observe/api-manager-analytics/analytics-pages/analytics-pages-latency).
  
## Error Rate Widget
![overview page latency]({{base_path}}/assets/img/observe/overview/overview-page-error-rate.png)

> This widget will display the average error rate (error count/total request count) in your selected environment for

Reviewer: ```suggestion
This widget displays the average error rate (error count/total request count) in your selected environment for
```

>  the selected time period. You can utilize this widget as an indicator to know the health of the system. If error

Reviewer: ```suggestion
 the selected time period. You can use this widget as an indicator to know the health of the system. If error
```

>   rate is high you will need to investigate more via 

Reviewer: ```suggestion
  rate is high, investigate further using the 
```

  [error page]({{base_path}}/observe/api-manager-analytics/analytics-pages/analytics-pages-erros).
  
## API Request Summary Timeline
![overview page latency]({{base_path}}/assets/img/observe/overview/overview-page-timeline.png)

> This chart combine above three widgets and plot them as a timeline. Left y-axis will show the 'request count' and 'error

Reviewer: ```suggestion
This chart combines the above three widgets and plots them in a timeline. The y-axis shows the 'request count' and 'error
```

>  count. x-axis displays time and right y-axis shows latency in milliseconds. Granularity of the data points will be

Reviewer: ```suggestion
 count. The x-axis displays time and right y-axis on the right shows latency in milliseconds. Granularity of the data points are
```

>   decided based on the time range you have selected. Tooltip will provide the exact value of all three metrics

Reviewer: ```suggestion
  decided based on the time range you have selected. The tooltip accurately provides the exact value of all three metrics.
```

>    accurately. 

Reviewer: ```suggestion
```
